src/components/Profile.tsx

```diff
@@ -8,40 +8,53 @@ import {
 
 import styles from './styles.module.css';
 import { AnyObject } from '../common';
+const ignoredNodeKeys = ['emailType', 'addressType', 'phoneType', '__typename'];
```

Comment:
Should `emailType`, `addressType` and `phoneType` be shown actually? Those are data that Profile BE provides to services. Perhaps this example service can show all available data?

Reply:
The value of those is always "OTHER". Other services than Profile UI might change it I guess, but at the moment the value is pointless?

Reply:
We don't know. Some services might rely on it. The data is anyway there on the schema and it's not marked "deprecated" nor "unreliable" or anything like that, so I think we should treat it as any other data.

Reply:
Changed. Only "__typename" is now filtered as it is useless in this app. If needed for some reason, it can be checked via console.